### PR TITLE
[WIP] Implemented Page.AddStyleTag tests

### PR DESCRIPTION
**Pull request by PhenX**

### CODE REVIEW

**Comment by kblok** [comment]
Thank you @PhenX for jumping in, I really appreciate it! 
A few things I would like to mention:
1. Remember to add `closes #<issue number>` in the PR description, so the issue moves in the project board

2. It's **very important** to follow our current [roadmap](https://github.com/kblok/puppeteer-sharp/projects). We are working now on version [v0.4](https://github.com/kblok/puppeteer-sharp/projects/7). And this PR is implementing https://github.com/kblok/puppeteer-sharp/issues/41 which is planned to v1.0.

So if you are close to finishing this PR go ahead and implement this. But please, let's follow the roadmap.

File: lib/PuppeteerSharp.Tests/Page/AddStyleTagsTests.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            await Page.GoToAsync(TestConstants.EmptyPage);
+
+            var styleHandle = await Page.AddStyleTagAsync(new AddTagOptions
+            {
+                Url = "/injectedstyle.css"
+            });
+            
+            Assert.NotNull(styleHandle.AsElement());
+
+            var bgColor = await Page.EvaluateExpressionAsync(BackgroundColorScript);
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/Page/AddStyleTagsTests.cs]: use `EvaluateExpressionAsync<string>` and remove the `ToString` on the next line

File: lib/PuppeteerSharp.Tests/Page/AddStyleTagsTests.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            
+            var styleHandle =
+                await Page.AddStyleTagAsync(new AddTagOptions
+                {
+                    Content = "body { background-color: green; }"
+                });
+
+            Assert.NotNull(styleHandle.AsElement());
+
+            var bgColor = await Page.EvaluateExpressionAsync(BackgroundColorScript);
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/Page/AddStyleTagsTests.cs]: use `EvaluateExpressionAsync<string>` and remove the `ToString` on the next line

**Comment by Meir017** [comment]
The tests need to pass...

**Comment by kblok** [comment]
@PhenX yeah, go ahead and implement it if you want. If not @Meir017 or myself can start from here. Having tests coded is also super important for us.

**Comment by kblok** [comment]
Yeah and that's not scheduled for v0.4

**Comment by kblok** [comment]
Let's create a new Pr when we get this done

**Comment by Meir017** [comment]
@PhenX are you up to update this?

**Pull request closed without merging**